src/lib/analytics/AnalyticsClass.js

```diff
@@ -0,0 +1,383 @@
+// @flow
+import { assign, debounce, forEach, get, isString, pick } from 'lodash'
+import { isMobileReactNative } from '../utils/platform'
+import { LogEvent } from '../logger/pino-logger'
+import { ExceptionCategory } from '../logger/exceptions'
+import { ERROR_LOG } from './constants'
+
+export class AnalyticsClass {
+  apis = {}
+
+  rootApi = null
+
+  loggerApi = null
+
+  secureTransmit = false
+
+  isSentryEnabled = false
+
+  isAmplitudeEnabled = false
+
+  isFullStoryEnabled = false
+
+  amplitudeKey = null
+
+  sentryDSN = null
+
+  version = null
+
+  network = null
+
+  env = null
+
+  phase = null
+
+  logger = null
+
+  loggerApi = null
```

Comment:
```suggestion
```

Reply:
i think it's a these are unnecessary declaration (but pls check aren't something broken after i removed this)